gazebo_ros/test/worlds/sdf_node_plugin.world

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<sdf version="1.6">
+  <world name="default">
+    <plugin name="sdf_plugin" filename="./libsdf_node.so">
+      <ros>
+        <node_name>my_node</node_name>
+        <namespace>/foo</namespace>
+        <argument>test:=my_topic</argument>
+        <parameter name="my_string" type="string">str</parameter>
+        <parameter name="my_bool_true" type="bool">True</parameter>
+        <parameter name="my_bool_false" type="bool">false</parameter>
+        <parameter name="my_int" type="int">42</parameter>
+        <parameter name="my_double" type="double">13.37</parameter>
```

Comment:
Suggestion: add a malformed parameter (for example missing type) and check that the parameter is not found afterwards.

Reply:
added a few failure examples